.github/workflows/release.yml

```diff
@@ -35,3 +35,24 @@ jobs:
         env:
           MAVEN_USERNAME: ${{ secrets.FR_ARTIFACTORY_USER }}
           MAVEN_CENTRAL_TOKEN: ${{ secrets.FR_ARTIFACTORY_TOKEN }}
+      
+      - name: helm version
+        id: helm
+        run: |
+          HELM_VERSION=$(make version)
+          echo ::set-output name=version::"$HELM_VERSION"
+
+      - name: package helm
+        run: |
+          make helm version=${{ steps.helm.outputs.version }}
+
+      - name: Publish Release Chart 
+        id: upload
+        uses: google-github-actions/upload-cloud-storage@main
+        with:
+          path: ${{ env.SERVICE_NAME }}-${{ steps.helm.outputs.version }}.tgz
+          destination: ${{ secrets.HELM_RELEASE_BUCKET }}/${{ env.SERVICE_NAME }}
+      
+      - name: Release Docker Image
+        run: |
+          make docker tag=${{ github.event.release.tag_name }} gcr-repo=${{ secrets.RELEASE_REPO }}
```

Comment:
Are we sure that the make command will push the tag latest too?

Reply:
Sorry that's ok, just I saw the commit `remove latest tag from pr and add it to merge`!